### PR TITLE
Added slides (with new images). Fixed typos in tutorial.

### PR DESCRIPTION
**Pull request by kxk302**
Added slides for FNN, with some new images. The content of slides has already been reviewed. So, I guess its just the format that needs to be approved. 

Also, fixed some typos in tutorial. 

Thank you!

### CODE REVIEW

File: topics/statistics/tutorials/FNN/slides.html [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+???
+
+What is an artificial neural network?
+
+---
+
+# Artificial Neural Networks
+
+- ML discipline roughly inspired by how neurons in a human brain work
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:37]: ML or DL?

File: topics/statistics/tutorials/FNN/slides.html [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+- Huge resurgence due to availability of data and computing capacity
+- Various types of neural networks (Feedforward, Recurrent, Convolutional)
+- FNN applied to classification, clustering, regression, and association 
+
+---
+
+# Inspiration for neural networks
+
+- Neuron a special biological cell with information processing ability 
+	- Receives signals from other neurons through its *denderites*
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:47]: dendrites?

File: topics/statistics/tutorials/FNN/slides.html [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+- Neurons communicate by signals ms in duration
+	- Signal transmission frequecny up to several hundred Hertz
+	- Millions of times slower than an electronic circuit
+	- Complex tasks like face recognition done within a few hundred ms
+	- Computation involved cannot take more than 100 serial steps
+- The information sent from one neuron to another is very small 
+	- Critical information not transmitted
+	- But captured by the interconnections
+- Distributed computation/representation of the brain
+	- Allows slow computing elements to perfrom complex tasks quickly
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:78]: perform

File: topics/statistics/tutorials/FNN/slides.html [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+# Output layer (Continued)
+
+- Multiclass classification
+	- As many neurons in output layer as number of classes
+	- *Softmax* activation function
+	- Takes input to neurons in output layer 
+	- Creates a probability distribution, sum of outputs adds up to 1 
+	- The neuron with the highest proability is the predicted label
+- Regression problem
+	- Sinfle neuron in output layer
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:172]: Single?

File: topics/statistics/tutorials/FNN/slides.html [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+---
+
+# Loss/Cost functions
+
+- During training, for each training example (x^i,y^i), we present x^i to neural network
+	- Compare predicted output with label y^1 
+	- Need loss function to measure difference between predicted & expected output
+- Use *Cross entropy* loss function for classification problems 
+- And *Quadratic* loss function for regression problems
+	- Qadratic cost function is also called *Mean Squared Error (MSE)*
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:184]: Quadratic?

File: topics/statistics/tutorials/FNN/slides.html [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+# Types of Gradient Descent
+
+- *Batch* gradient descent
+	- Calculate gradient for each weight/bias for *all* samples
+	- Average gradients and update weights/biases
+	- Slow, if we have too many samples  
+
+- *Stochastic* gradient descent
+	- Update weights/biases based on gradient of *each* sample
+        - Fast. Not accurate if sample gradient not representiative
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:237]: Is the indentation correct for `- Fast`?

File: topics/statistics/tutorials/FNN/slides.html [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+---
+
+# Backpropagation (BP) learning algorithm
+
+- A *gradient descent* technique
+	- Find local minimum of a function by iteratively moving in opposite direction of gradient of function at current point 
+- Goal of learning is to minimize cost function given training set 
+	- Cost function is a function of network weights & biases of all neurons in all layers 
+	- Backpropagation iteratively computes gradient of cost function relative to each weight and bias
+	- Updates weights and biases in the opposite direction of gradient
```
Comment (anuprulez) [inline topics/statistics/tutorials/FNN/slides.html:211]: Maybe adding one more point would be useful?
- Gradients (partial derivatives) are used to update weights and biases.

**Review by anuprulez — CHANGES_REQUESTED** [review]

File: topics/statistics/tutorials/FNN/slides.html (hunk 1 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:37]: I'm old school, so ML :)

File: topics/statistics/tutorials/FNN/slides.html (hunk 2 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:47]: Fixed

File: topics/statistics/tutorials/FNN/slides.html (hunk 3 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:78]: Fixed

File: topics/statistics/tutorials/FNN/slides.html (hunk 4 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:172]: Fixed

File: topics/statistics/tutorials/FNN/slides.html (hunk 5 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:184]: Fixed

File: topics/statistics/tutorials/FNN/slides.html (hunk 6 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:237]: No, fixed it. Thanks.

File: topics/statistics/tutorials/FNN/slides.html (hunk 7 above)
Comment (kxk302) [reply topics/statistics/tutorials/FNN/slides.html:211]: Added a bullet.

**Comment by kxk302** [comment]
Revisions are pushed. Thanks @anuprulez

**Review by hexylena — APPROVED** [review]

**Comment by kxk302** [comment]
Hi @anuprulez 
Could you please merge this if it looks ok now?
Thanks!

**Review by hexylena — COMMENTED** [review]
apparently I never made this comment since I accidentally clicked 'start review' instead of 'make comment'

File: topics/statistics/tutorials/FNN/workflows/Intro_To_FNN.ga [hunk 8]
```diff
@@ -166,15 +166,15 @@
                 "top": 10
             },
             "post_job_actions": {},
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/keras_model_builder/keras_model_builder/0.5.0",
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/keras_model_builder/keras_model_builder/0.4.2",
```
[the remark this answers is not in the document]
Comment (kxk302) [reply topics/statistics/tutorials/FNN/workflows/Intro_To_FNN.ga:169]: Updated the tutorial as well. Thanks for catching this.

**Review by hexylena — APPROVED** [review]

**Comment by bgruening** [comment]
@anuprulez is busy the next days. This looks good to me and I think is a great first version. We can adjust slides until GCC if we wish. Thanks, everyone.

**Comment by hexylena** [comment]
Thanks @bgruening good to know so we aren't waiting!

**Comment by kxk302** [comment]
thank you all!

**Pull request merged**